lib/venue.rb

```diff
@@ -1,2 +1,19 @@
 class Venue
+  attr_reader :name, :capacity, :patrons
+
+  def initialize (venue, capacity)
+    @name = venue
+    @capacity = capacity
+    @patrons = []
+  end
+
+  def add_patron(patrons)
+    @patrons = ['Mike', 'Megan', 'Bob']
+
+  end
+
+  def yell_at_patrons(patrons)
```

Comment:
I don't think that the `yell_at_patrons` takes an argument. 
```
venue.yell_at_patrons
=> ["MIKE", "MEGAN", "BOB"]
```
Remember that the `=>` is showing you what the return value should be. 

So you will want to iterate through your `@patrons` array and then `.upcase` the `block_variable`. Revisit the `#each` lesson (https://backend.turing.edu/module1/lessons/each)